### ISSUE TITLE
Niekompletne dane w historii przesyłki.

### ISSUE THREAD

**Issue by kjonca** [allegro/allegro-api#9724]
Występuje problem: 
 - [X] z zasobem REST API / with a REST API resource
 - [ ] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**
curl -X GET -H "authorization: Bearer ..." -H "Accept: application/vnd.allegro.public.v1+json" 'https://api.allegro.pl/order/carriers/ALLEGRO/tracking?waybill=....'
W odpowiedzi jest(m.in.)
`{"occurredAt":"2024-08-23T16:40:19Z","code":"IN_TRANSIT","description":"Przesyłka została przyjęta w oddziale"}`
tymczasem w podglądzie allegro mam:

==========
23 sie 2024, 18:40
Przesyłka została przyjęta w oddziale
WARSZAWA
==========

Jak mogę uzyskać owo "Warszawa"  w api ?

**Comment by MaciejFrackowiak** [comment]
Proszę podaj trace-id wywołania z API oraz w miarę możliwość zamieść zrzut ekranu z widoku WWW.

**Comment by kjonca** [comment]
![image](https://github.com/user-attachments/assets/156c2e89-572d-4914-bd16-1e07969170bf)
Nie wiem o co chodzi z tym trace-id - nie widzę w odpowiedzi. 
Wołam to tak (to moja przesyłka, ale jeśli miałoby to spowodować zalew odpytań to możecie wyciąć numer):
`curl -vv -X GET -H "authorization: Bearer  qqrq" -H "Accept: application/vnd.allegro.public.v1+json" 'https://api.allegro.pl/order/carriers/ALLEGRO/tracking?waybill=A000ZLEZV2'`

```
{"carrierId":"ALLEGRO","waybills":[{"waybill":"A000ZLEZV2","trackingDetails":{"createdAt":"2024-08-23T10:30:13.623Z","updatedAt":"2024-08-24T16:48:40.532Z","statuses":[{"occurredAt":"2024-08-23T10:30:13.623Z","code":"PENDING","description":"Przesyłka została przygotowana przez nadawcę"},{"occurredAt":"2024-08-23T11:14:21Z","code":"IN_TRANSIT","description":"Przesyłka została odebrana przez kuriera"},{"occurredAt":"2024-08-23T16:40:19Z","code":"IN_TRANSIT","description":"Przesyłka została przyjęta w oddziale"},{"occurredAt":"2024-08-24T06:04:37Z","code":"RELEASED_FOR_DELIVERY","description":"Przesyłka została wydana do doręczenia"},{"occurredAt":"2024-08-24T12:26:03Z","code":"ISSUE","description":"Awaria maszyny"},{"occurredAt":"2024-08-24T12:26:17.238Z","code":"IN_TRANSIT","description":"Przesyłka została przekierowana do maszyny"},{"occurredAt":"2024-08-24T12:26:48.985Z","code":"AVAILABLE_FOR_PICKUP","description":"Przesyłka oczekuje na odbiór"},{"occurredAt":"2024-08-24T16:48:40.532Z","code":* Connection #0 to host api.allegro.pl left intact
"DELIVERED","description":"Przesyłka została doręczona"}]}}]}
```

**Comment by MaciejFrackowiak** [comment]
Dziękuję za zrzut i przykład. Widoczność przy statusie miasta to cecha dostępna wyłącznie w serwisie WWW. Obecny zasób API nie posiada tej funkcjonalności.

**Comment by kjonca** [comment]
A jest planowane? Bo trochę dziwnie to wygląda.

**Comment by MaciejFrackowiak** [comment]
Nie, nie ma tego w najbliższych planach. Jeśli zależy Ci na tej zmianie, możesz [zgłosić](https://github.com/allegro/allegro-api/discussions) sugestię zmiany.
